test/token-verification.test.js

```diff
@@ -288,3 +288,52 @@ describe('jwt-verification', function () {
     });
   });
 });
+describe('access_token validation', function() {
+  it('should handle empty/null access tokens', function(done) {
+    var access_token = null;
+    var alg = 'RS256';
+    var at_hash = 'at_hash';
+
+    var itv = new IdTokenVerifier();
+    itv.validateAccessToken(access_token, alg, at_hash, function(err) {
+      expect(err.name).to.be('TokenValidationError');
+      expect(err.message).to.be('Invalid access_token');
+      done();
+    });
+  });
+  it('should throw an errr whith HS256 id_token', function(done) {
```

Comment:
errr -> error
whith -> with 
got caipirinhas? 😂

Reply:
it's a pirate error. errr

Reply:
😂 😂 😂 😂 😂 😂 😂

Reply:
☠️